stetho-sample/src/debug/java/com/facebook/stetho/sample/SampleDebugApplication.java

```diff
@@ -42,7 +49,56 @@ private void initializeStetho(final Context context) {
                 .finish();
           }
         })
-        .enableWebKitInspector(Stetho.defaultInspectorModulesProvider(context))
+        .enableWebKitInspector(new ExtInspectorModulesProvider(context))
         .build());
   }
+
+  private static class ExtInspectorModulesProvider implements InspectorModulesProvider {
+
+    private Context mContext;
+
+    ExtInspectorModulesProvider(Context context) {
+      mContext = context;
+    }
+
+    @Override
+    public Iterable<ChromeDevtoolsDomain> get() {
+      return new Stetho.DefaultInspectorModulesBuilder(mContext)
+          .provideDatabaseDriver(createContentProviderDatabaseDriver(mContext))
```

Comment:
nit: prefer less verbose method names, such as `databaseDriver`.  Note the others: `documentProvider`, `databaseFiles`, etc.